components/widget/main_page/widget_main_page_transfers_list.lua

```diff
@@ -0,0 +1,97 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Widget/MainPage/TransfersList
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+
+local TransferList = Lua.import('Module:TransferList')
+
+local Widget = Lua.import('Module:Widget')
+local HtmlWidgets = Lua.import('Module:Widget/Html/All')
+local Div = HtmlWidgets.Div
+local Link = Lua.import('Module:Widget/Basic/Link')
+local Span = HtmlWidgets.Span
+local WidgetUtil = Lua.import('Module:Widget/Util')
+
+local CENTER_DOT = Span{
+	css = {
+		['font-style'] = 'normal',
+		padding = '0 5px',
+	},
+	children = { '&#8226;' }
+}
+
+---@class TransfersListParameters
+---@field limit integer?
+---@field rumours boolean?
+---@field transferPortal string?
+---@field transferPage fun():string
+---@field transferQuery boolean?
```

Comment:
```suggestion
---@field transferQuery boolean?
---@field onlyNotableTransfers boolean?
```

Reply:
possible implementation via #5546 (stacked on this PR so if @ElectricalBoy likes it he can just merge it into this PR)